src/run_tribler.py

```diff
@@ -88,7 +88,13 @@ async def start_tribler():
     get_event_loop().run_forever()
 
 
+def excepthook(exctype, value, _):
+    SentryReporter.send_exception_with_confirmation(f'{exctype}: {value}.')
```

Comment:
It is possible that `SentryReporter.send_exception_with_confirmation()` will be called before `SentryReporter.init`, if the exception happens inside `SentryReporter.init()`. I think it should be handled. At least it is possible to set up some flag at the end of `SentryReporter.init()` that the reporter is properly initialized and check this flag inside `SentryReporter.send_exception_with_confirmation()`

Reply:
Yes, it is possible that `SentryReporter.init` will fail.
But if `SentryReporter.init` fails it will not be possible to send us the event.
Then no reason to handle this situation.